src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -42,32 +42,26 @@ Viewport Terminal::GetBufferSize() noexcept
     return _buffer->GetSize();
 }
 
-bool Terminal::SetCursorPosition(short x, short y) noexcept
+bool Terminal::SetCursorPosition(til::point pos) noexcept
 try
 {
     const auto viewport = _GetMutableViewport();
-    const auto viewOrigin = viewport.Origin();
-    const short absoluteX = viewOrigin.X + x;
-    const short absoluteY = viewOrigin.Y + y;
-    COORD newPos{ absoluteX, absoluteY };
+    const til::point viewOrigin{ viewport.Origin() };
+    auto newPos = til::unwrap_coord(viewOrigin + pos);

```

Comment:
This commit contains quite a number of explicit wrapping and unwrapping of COORD/SMALL_RECT into til::point/til::rect, or explicit narrowings into SHORTs, because only a part of the project was converted to the til helpers. In a future PR these changes will be undone, since barely anything will use COORD/SMALL_RECT then.